### PR TITLE
schismtracker: url updates

### PR DESCRIPTION
**Pull request by vszakats**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [ ] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

### CODE REVIEW

File: Formula/schismtracker.rb [hunk 1]
```diff
@@ -11,16 +12,11 @@ class Schismtracker < Formula
     sha256 "f892d60e3e74f352d7d9efbbf84b6470e4e751ffafe55c9b6ab9913bd611dba2" => :mavericks
   end
 
-  head do
-    url "http://schismtracker.org/hg/", :using => :hg
-
-    depends_on "autoconf" => :build
-  end
-
+  depends_on "autoconf" => :build
```
Comment (UniqMartin) [inline Formula/schismtracker.rb]: Like I mentioned in the other PR, `autoconf` is actually only used by the HEAD build, so it should stay in the `head do` block, just like the HEAD `url` should.

Comment (vszakats) [inline Formula/schismtracker.rb]: Isn't the new source archive created from a source tag?

I'll revert this, but the build failed last time with the new source archive.

**Comment by vszakats** [comment]
Reverted. Let's see the CI results.

**Comment by UniqMartin** [comment]
I was hoping you could try and build that locally to see what changes are needed in the `install` method to accommodate the changes caused by the relocated archive.

It seems my code comment didn't make sense and you actually instinctively/accidentally did the right thing. Sorry for making you revert that! There's no longer a `configure` in the archive, so we will have to run `autoreconf` unconditionally. I think replaceing (in `def install`):

```ruby
system "autoreconf", "-ivf" if build.head?
```

with

```ruby
system "autoreconf", "-ivf"
```

could do the trick.

File: Formula/schismtracker.rb (hunk 1 above)
Comment (UniqMartin) [inline Formula/schismtracker.rb]: Yes, you're right. (See my other non-code comment below.)

**Comment by vszakats** [comment]
Oops, nice find, I missed that. Let's see how it works after that.

**Comment by UniqMartin** [comment]
Build failed, looks like adding `depends_on "automake" => :build` should fix that.

**Comment by UniqMartin** [comment]
Merged in 470e7aabade01e56cc0ae4a36e19e258c4b035ca. Thank you for yet another contribution to Homebrew, @vszakats! :tada:

**Pull request closed without merging**